### PR TITLE
Fixes build errors related from tests

### PR DESCRIPTION
**Pull request by akhila-raju**
## PR Description
Fixes build errors from test files

### CODE REVIEW

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 1]
```diff
@@ -1264,7 +1270,7 @@ public static void activate_validator(BeaconState state, int index, boolean is_g
    */
   private static int ceil_div8(int div) {
```
Comment (cemozerr) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: Is this function being used anymore?

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 2]
```diff
@@ -817,10 +819,10 @@ public static int get_permuted_index(int index, int list_size, Bytes32 seed) {
 
       // This needs to be unsigned modulo.
       int pivot =
-          (int)
```
Comment (cemozerr) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:820]: Is there a reason to remove toIntExact? it seems safer than (int)

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java [hunk 3]
```diff
@@ -74,7 +74,7 @@ public static boolean is_active_validator(Validator validator, UnsignedLong epoc
    */
   public static List<Integer> get_active_validator_indices(
       List<Validator> validators, UnsignedLong epoch) {
-    List<Integer> active_validator_indices = Collections.synchronizedList(new ArrayList<Integer>());
+    List<Integer> active_validator_indices = Collections.synchronizedList(new ArrayList<>());
```
Comment (dangerousfood) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java:77]: Is there a design reason for using an undefined generic?

**Review by dangerousfood — APPROVED** [review]

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java (hunk 2 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:820]: This change shows (int) being removed and toIntExact being added :)

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java (hunk 3 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/ValidatorsUtil.java:77]: It's an IntelliJ suggestion

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java (hunk 1 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: It's not, I will remove it!]

**Pull request merged**